bin/_tag.sh

```diff
@@ -19,12 +19,12 @@ dir_tag() {
     echo "git-$(git log -n 1 --format="%h" "$dir")"
 }
 
-clean_head_root_tag() {
+head_root_tag() {
+    sha=$(git_sha HEAD)
     if git diff-index --quiet HEAD -- ; then
-        echo "git-$(git_sha HEAD)"
+        echo "git-$sha"
     else
-        echo "Commit unstaged changes or set an explicit build tag." >&2
-        exit 3
+        echo "$USER-$sha"
```

Comment:
so, since it's technically possible that we eventually work with someone who prefers to go by "git", perhaps better to use a dedicated non-git prefix and suffix with the username...

Reply:
yeah that had occurred to me then i thought whoever named themselves `git` deserved it. ;) i'll make the change.